ci/install-musl.sh

```diff
@@ -68,6 +68,13 @@ case ${1} in
           ./configure --prefix="/musl-${musl_arch}" --enable-wrapper=yes
         make install -j4
         ;;
+    powerpc64*)
+	musl_arch=powerpc64
+	kernel_arch=powerpc
+	CC=${1}-linux-gnu-gcc CFLAGS="-mlong-double-64" \
+	  ./configure --prefix="/musl-${musl_arch}" --enable-wrapper=yes
+	make install -j4
+	;;
```

Comment:
Quote `"${1}-linux-gnu-gcc"` just so we don't have to think about splitting, could you also match the indentation of the other arms?

Also, shouldn't long double either be ibm double-double or binary128?

Reply:
`The default long double format is to use IBM extended double. Until all of the libraries are converted to use IEEE 128-bit floating point, it is not recommended to use --with-long-double-format=ieee.`